src/App.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
```

Comment:
As i know in vue 3 isn't needed wrapped element
https://v3.vuejs.org/guide/migration/fragments.html#_2-x-syntax